tests/Unit/PointwiseFunctions/AnalyticSolutions/Elasticity/BentBeam.py

```diff
@@ -17,6 +17,18 @@ def displacement(x, length, height, bending_moment, bulk_modulus,
     ])
 
 
+def strain(x, length, height, bending_moment, bulk_modulus, shear_modulus):
+    youngs_modulus = 9. * bulk_modulus * shear_modulus / \
+        (3. * bulk_modulus + shear_modulus)
+    poisson_ratio = (3. * bulk_modulus - 2. * shear_modulus) / \
+        (6. * bulk_modulus + 2. * shear_modulus)
+    prefactor = 12. * bending_moment / (youngs_modulus * height**3)
```

Comment:
The Young's modulus and poisson ratio relations to bulk and shear come up a lot, and are not specific to the bent beam...these should be in PointwiseFunctions

Reply:
Thanks for adding these generic functions to your python code...could you also please add c++ versions in src/PointwiseFunctions/Elasticity?

Reply:
There is functions within IsotropicHomogeneous to return the Young's modulus and Poisson's ratio, since they are specific to an isotropic homogeneous constitutive relation. I have a PR for the constitutive relation of a cubic crystal, where these do not hold true. Should I add them in a file separate to IsotropicHomogeneous itself?

Reply:
These functions are specific to the isotropic homogeneous constitutive relation, so I think they are fine where they are. Looks like they are not being tested though. Let's add tests to `Test_IsotropicHomogeneous.cpp` that call the Python functions you just added Tom, but that should probably be a separate PR. I created https://github.com/sxs-collaboration/spectre/issues/2257 and assigned myself. @geoffrey4444 does that work for you?